content/developers/office_hours.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Community Office Hours
+type: documentation
+---
+
+## Community Code Review
+
+Datadog runs twice monthly code reviews and office hours to assist community members who are contributing to our [open source projects](#list-of-open-source-project).  
+
+**These sessions are held on the 2nd and 4th Wednesday of each month.**
+
+### Where
+
+Office hours will be held in #integrations in the [Datadog Community Slack][1] ([get an invitation][2]), in conjunction with a [Google Hangout][3].
+
+### When
+
+* 2nd Wednesday of each month at 10:00am eastern time (2:00pm UTC).  
+* 4th Wednesday of each month at 3:00pm eastern time (7:00pm UTC).
+
+### Guidelines and Caveats
+
+- Review and feedback during office hours is offered on a first come first served basis. We may not get to every review each week. We appreciate your patience and understanding.
+
+- We welcome everyone from the community to join us and provide respectful and constructive feedback to their fellow members of the Datadog community. 
+
+- These sessions are not intended as a channel for general Datadog support. We are primarily focused on discussing and reviewing `datadog-agent` integrations, and contributions to other Datadog [open source projects](#list-of-open-source-project). For product support we encourage you to reach out to [our support team directly][4].
+
+- Sometimes a given PR maybe complex and require further offline review before we can provide feedback.  
+
+## List of open source project
```

Comment:
Imho this list should be split out to a sub-section.

Reply:
I tried with one list, and it was a little bit messy.. hence this sub-section division. But i'm totally open to change the layout if needed.